package.json

```diff
@@ -1,10 +1,11 @@
 {
   "name": "css-declaration-sorter",
-  "version": "7.0.0",
+  "version": "7.0.1",
   "description": "Sorts CSS declarations fast and automatically in a certain order.",
   "type": "module",
   "main": "./dist/main.cjs",
   "exports": {
+    "types": "./src/main.d.ts",
```

Comment:
Latest versions of TS require the `types` key to be present if `exports` is used.

Reply:
Could you share a link to docs that describe using it at `exports.types`?

https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html
Only shows it in the package.json 'root', which is set already.

https://www.typescriptlang.org/docs/handbook/esm-node.html
Only shows it under `exports._X_.types`.